Sources/XcodeGraphMapper/Mappers/Project/PBXProjectMapper.swift

```diff
@@ -175,7 +175,7 @@ struct PBXProjectMapper: PBXProjectMapping {
                let pathString = try file.fullPath(sourceRoot: xcodeProj.srcPathString)
             {
                 let path = try AbsolutePath(validating: pathString)
-                if try await fileSystem.exists(path, isDirectory: true),
+                if (try? await fileSystem.exists(path, isDirectory: true)) ?? false,
```

Comment:
fly-by fix: if `path` is not valid, we'd throw here. Instead, we default to `false` now